modules/apps/29-fee/keeper/grpc_query_test.go

```diff
@@ -769,14 +836,19 @@ func (suite *KeeperTestSuite) TestQueryFeeEnabledChannel() {
 			suite.SetupTest() // reset
 			expEnabled = true
 
-			suite.path.Setup()
+			path = ibctesting.NewPath(suite.chainA, suite.chainB)
+			enableFeeOnChannel(path)
+			path.EndpointA.ChannelID = ibctesting.FirstChannelID
 
 			req = &types.QueryFeeEnabledChannelRequest{
-				PortId:    suite.path.EndpointA.ChannelConfig.PortID,
-				ChannelId: suite.path.EndpointA.ChannelID,
+				PortId:    path.EndpointA.ChannelConfig.PortID,
+				ChannelId: path.EndpointA.ChannelID,
 			}
 
 			tc.malleate()
+			path.Setup()
```

Comment:
Cannot you move this line to replace `path.EndpointA.ChannelID = ibctesting.FirstChannelID` on line 841? Then you could do the following:

```go
{
  "fee not enabled on channel",
  func() {
    expEnabled = false
    path = ibctesting.NewPath(suite.chainA, suite.chainB)
    path.Setup()

    req = &types.QueryFeeEnabledChannelRequest{
      PortId:    path.EndpointA.ChannelConfig.PortID,
      ChannelId: path.EndpointA.ChannelID,
    }
  },
  true,
},
```

so you don't need to make `path` a common variable. 

Happy also to wait for other people's opinion.

Reply:
yes, that would work execpt for the test `"fee not enabled on channel"`. In that test we disable the fee by reachin inside path and doing `path.EndpointA.ChannelConfig.Version = ibcmock.Version`, so if we call `path.Setup()` before `tc.malleate()` this would do nothing and the fee would stay enabled. There's probably a way to achieve both but I am not familiar enough with the code here to find it :)

Reply:
I tested locally with the changes for `fee not enabled on channel` and it seemed to work, because we are doing the request for a new channel that doesn't have fee enabled. Let me know if I'm missing something though!

Reply:
whoops I totally misread your code snippet and did something different! Thanks for testing!